gocd/xcdchk.jump.opensuse.yaml

```diff
@@ -0,0 +1,21 @@
+format_version: 3
+pipelines:
+  openSUSE.QA.Maint:
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-xcdchk
+    materials:
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    timer:
+      spec: 0 */10 * ? * *
+      only_on_changes: false
+    stages:
+    - Run:
+        approval: manual
+        resources:
+          - staging-bot
```

Comment:
staging-bot is for quick jobs. this sounds like an expensive job and as such should be in the repo-checker resource

Reply:
Fixed